torch/csrc/api/include/torch/nn/modules/sequential.h

```diff
@@ -90,6 +90,66 @@ namespace nn {
 /// \endrst
 class SequentialImpl : public Cloneable<SequentialImpl> {
  public:
+  class NamedSubmodule {
```

Comment:
For why we need to have this class, see https://github.com/pytorch/pytorch/pull/17552/files#diff-ad62a46b3133f7412c5667d4c194359eR395.